runner/src/server/plugins/engine/components/MonthYearField.ts

```diff
@@ -66,10 +68,17 @@ export class MonthYearField extends FormComponent {
 
   getDisplayStringFromState(state: FormSubmissionState) {
     const values = state[this.name];
-    const month = values[`${this.name}__month`] ?? "Not supplied";
-    const year = values[`${this.name}__year`] ?? "Not supplied";
+    const year = values?.[`${this.name}__year`] ?? "Not supplied";
 
-    return [month, year].join(" / ");
+    let monthString = "Not supplied";
+    const monthValue = values?.[`${this.name}__month`];
+    if (monthValue) {
+      const date = new Date();
+      date.setMonth(values[`${this.name}__month`]);
+      monthString = date.toLocaleString("default", { month: "long" });
+    }
+
+    return `${monthString} ${year}`;
```

Comment:
We should be displaying back to the user what they entered, so would expect this to be '11/2021', not 'November 2021'

Reply:
check your answers pattern guidance displays month (long), rather than what the user entered. I haven't changed the smoke tests either. Smoke tests are testing for "22 March 2021" smoke-tests/designer/features/steps/completeAForm.steps.js

Content style guide also suggests this https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#dates

https://design-system.service.gov.uk/patterns/check-answers/

Reply:
do you still want me to change this @superafroman?